setup.py

```diff
@@ -130,6 +133,7 @@ def run(self):
         'doc': doc,
         'docker': docker,
         'druid': druid,
+        'gcs': gcs,
```

Comment:
Curious: is this introducing a new airflow[gcs] extra package? Up to now, the Google contrib stuff has come as part of the main airflow package.